cli/check_config_test.go

```diff
@@ -0,0 +1,17 @@
+package cli
+
+import (
+	"testing"
+)
+
+func TestCheckConfig(t *testing.T) {
+	err := CheckConfig([]string{"testdata/conf.good.yml"})
+	if err != nil {
+		t.Fatalf("error: %v", err)
```

Comment:
Instead of `error: ` how about `checking valid config file failed with:  `?

Reply:
done